superset/migrations/versions/2022-06-27_14-59_7fb8bca906d2_permalink_rename_filterstate.py

```diff
@@ -87,5 +86,3 @@ def downgrade():
                 state["hash"] = state["anchor"]
                 del state["anchor"]
             entry.value = pickle.dumps(value)
-        session.merge(entry)
```

Comment:
There's no need to merge the existing entry. See [here](https://michaelcho.me/article/sqlalchemy-commit-flush-expire-refresh-merge-whats-the-difference) for details:

> Used when you may have more than 1 in-memory objects which map to the same database record with some key.